server/statistics/store_collection.go

```diff
@@ -238,10 +244,10 @@ type storeStatisticsMap struct {
 }
 
 // NewStoreStatisticsMap creates a new storeStatisticsMap.
-func NewStoreStatisticsMap(opt *config.PersistOptions) *storeStatisticsMap {
+func NewStoreStatisticsMap(opt *config.PersistOptions, manager *config.StoreConfigManager) *storeStatisticsMap {
```

Comment:
How about only passing `*config.StoreConfig`?

Reply:
In past,  the  point of the `*config.StoreConfig` will be replace not the value after config be updated. I will change it in next pr.